tuxemon/core/components/event/conditions/npc.py

```diff
@@ -299,13 +298,10 @@ def facing_npc(self, game, condition):
 
         # First, find the NPC by name
         world = game.current_state
-        for item in world.npcs:
-            if item.name == npc_name:
-                npc = item      # We found the NPC!
-
-        # If we couldn't find the NPC, return false as we're not next to it :P
-        if not npc:
-            return False
+        if npc_name not in world.npcs
+            return
+        else
+            npc = world.npcs[npc_slug]
 
```

Comment:
This bit was merged into develop after I started and I didn't catch it after merging from upstream. I'll get it fixed.